src/main/datalog_console/client.cljs

```diff
@@ -33,6 +35,23 @@
     (.postMessage port #js {:name ":datalog-console.client/init" :tab-id current-tab-id}))
   (catch js/Error _e nil))
 
+(defn db-actions []
+  (let [action (r/atom :entity)
```

Comment:
Maybe call this action-tab?

Reply:
I'd go with `active-tab`

Reply:
And maybe rename the whole component to `tabs`. It might be just me but `db-actions` was not intuitive. And if we add more "tabs" to the client later we can disambiguate then